.github/workflows/check-for-new-cascadia.yml

```diff
@@ -6,25 +6,45 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2        
+    - uses: octokit/request-action@v2.x
+      id: get_latest_release
       with:
-        token: ${{ secrets.DELUGIA_ACCESS_TOKEN }}
-        fetch-depth: 0
-    - name: Magic 🐇
-      run: |
-        # Find the latest Cascadia tag
-        CASCADIATAG=`curl --silent "https://api.github.com/repos/microsoft/cascadia-code/releases/latest" | jq -r .tag_name`
-        echo "Latest Cascadia tag is ${CASCADIATAG}"    
-        
-        # Check if we already have a matching tag
-        if git rev-parse "$CASCADIATAG" >/dev/null 2>&1; then
-          echo "${CASCADIATAG} tag exists";
-        else
-          echo "${CASCADIATAG} tag does not exist, creating it ..."
-          git config user.name github-actions
-          git config user.email github-actions@github.com
-          git tag -a ${CASCADIATAG} -m "Bump Cascadia version to ${CASCADIATAG}"    
-          git push origin --tags
-        fi  
+        route: GET /repos/{owner}/{repo}/releases/latest
+        owner: microsoft
+        repo: cascadia-code
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Find the latest Cascadia tag
```

Comment:
You could merge this step with the next one to get the latest tag and echo it in one step

Reply:
Makes sense.